crates/bevy_pbr/src/material.rs

```diff
@@ -309,6 +315,27 @@ where
     }
 }
 
+pub fn extract_visible_material_instances<M: Material>(
+    mut material_instances: ResMut<RenderMaterialInstances<M>>,
+    materials: Res<RenderAssets<PreparedMaterial<M>>>,
+    query: Extract<Query<(Entity, &ViewVisibility, &Handle<M>)>>,
+    mut pending_render_assets: ResMut<PendingRenderAssets<PreparedMaterial<M>>>,
+) {
+    material_instances.clear();
+    for (entity, view_visibility, material) in &query {
+        if view_visibility.get() {
```

Comment:
This isn't a change from how it is now, but meshlets uses RenderMaterialInstances. But, ViewVisibility should never get computed for MeshletMeshes?? It only gets updated for WithMesh entities, and it defaults to false afaiu. What am I missing here?

Reply:
Hmm, the meshlet example works, so there doesn't seem to be any problem. I'm not sure how it works, but this also didn't really change anything versus main, other than to allow use of `RenderAssetKey`.